### DISCUSSION TITLE
Show CC: Is this a good place?

### DISCUSSION THREAD

**Discussion by verdverm** [cuelang/cue#410] [General]
In the vein of "Show HN" and the community showing off what they are working on...

Is this an acceptable place for such announcements?

Is the `Show CC: ...` (and similar) a means to differentiate post intent and something we'd like to formalize?

**Comment by verdverm** [comment]
golang-nuts was a place for this early on

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/410.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
